RetailSuiteAccounting/UserInterface/02a_DebtorsCreditors.md

```diff
@@ -0,0 +1,2394 @@
+
+**Bank details**
+
+- *Account holder*  
+Enter the account holder name.
+
+- *Account*  
+Enter the customer/supplier bank account.
+
+- *Routing number*  
+Enter the bank routing number.
+
+- *Bank*  
+The bank name is filled in automatically when entering the routing number.
+
+- *BIC*  
+Enter the BIC (Bank Identifier Code).
+
+- *IBAN*  
+Enter the IBAN (International Bank Account Number).
+
+- *SEPA mandate reference*  
+Enter the SEPA mandate reference, if available.
+
+- *Signature date*  
+Enter the signature date or click the ![Calendar](../../Assets/Icons/Calendar.png "[Calendar]") (Calendar) button to select the date.
+
+
+**Payment options**
+
+- *Price group*  
+Click the drop-down list to select the appropriate option. All available price groups are displayed in the list.
+
+  > [Info] Price groups can be created in the *Settings* menu entry of the *Material management* and the *Order management* modules.
+
+- *Print gross*  
+Click the drop-down list to select the print mode. The following options are available:   
+
+  - **Net prices on receipts**
+  - **Gross prices on receipts**
+  - **Gross prices stating net total amount**
+
+
+**Tax**
+
+- *Tax office no.*  
+Enter the tax office number, if known. Alternatively, use the [SEARCH] button to search for the competent tax office.
+
+- [SEARCH]  
+Click this button to add a tax office. The *Select tax office* window is displayed, see [Select tax office](#select-tax-office).  
+
+- *Tax office*  
+Enter the competent tax office, if known.
+
+- *Tax no.*  
+Enter the customer/supplier tax number, if applicable.
+
+- *VAT ID*  
+Enter the customer/supplier VAT identification number, if applicable.
+
+
+**Posting**  
+
+- *Standard account*  
+Enter a standard account number for the postings.   
+
+- *Standard cost 1*  
+Enter a standard cost center for the postings.
+
+- *Standard cost 2*  
+Enter a standard cost unit for the postings.
+
+
+**Credit card details**
+
+[comment]: <> (im Credit card data)
+
+- *Card type*  
+Click the drop-down list to select the applicable card type.
+
+- *Card holder*  
+Enter the card holder name.
+
+- *Card number*  
+Enter the card number.
+
+- *Valid Thru*  
+Enter the card expiry date.
+
+
+**Default values**
+
+[comment]: <> (im System Standard values)
+
+- *Method*  
+Click the drop-down list to select the applicable method.
+
+- *Payment terms*  
+Click the drop-down list to select the applicable payment terms. The following options are available:  
+
+  - **1 - 8 d 2%, 30 d net**
+  - **2 - 30 d net**  
+
+
+- *Only cost supplier/ordering process inactive [x]*  
+Select or deselect this checkbox as necessary.
+
+  > [Info] This option is only relevant when managing supplier receipts. Select it if the supplier is a provider delivering a service, for example an internet provider, but not goods. If the checkbox is activated, the supplier will not be taken into account in the creation of order proposals.  
```

Comment:
```suggestion
- [x] *Only cost supplier/ordering process inactive*    
   Select this checkbox if the supplier is a provider delivering a service, for example an internet provider, but not goods. If the checkbox is selected, the supplier will not be taken into account in the creation of order proposals. Otherwise, deselect the checkbox. 

  > [Info] This option is only relevant when managing supplier receipts. 
```